scripts/installers/Dockerfile.ngramtools

```diff
@@ -0,0 +1,13 @@
+# How to use? Build it from NeMo root folder:
+# 1. git clone https://github.com/NVIDIA/NeMo.git && cd NeMo
+# 2. DOCKER_BUILDKIT=1 docker build -t nemo:22.12.2 -f ./scripts/installers/Dockerfile.ngramtools .
+
+from nvcr.io/nvidia/nemo:22.12
+
+WORKDIR /workspace/nemo
```

Comment:
Is it necessary to assume this location for the nemo?
How about passing it as the argument when running the script?

Reply:
This location comes from our main Docker file https://github.com/NVIDIA/NeMo/blob/main/Dockerfile#L109  Current scripts depends on that, and it will not work with other location

Reply:
So we are going to install the KenLM in our docker by default?
How the docker file runs this file?
What happens to users who don't use our dockers?

Reply:
Yes, entire script scripts/installers/Dockerfile.ngramtools is optional and only for people who need ngram rescoring on the top of NeMo